packages/react-core/src/components/Form/examples/FormFieldGroups.tsx

```diff
@@ -0,0 +1,388 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  FormFieldGroup,
+  FormFieldGroupExpandable,
+  FormFieldGroupHeader,
+  TextInput,
+  Button
+} from '@patternfly/react-core';
+import TrashIcon from '@patternfly/react-icons/dist/esm/icons/trash-icon';
+
+export const FieldGroups: React.FunctionComponent = () => {
+  const initialValues = {
+    'form-expandable-field-groups-label1': '',
+    'form-expandable-field-groups-label2': '',
+    'form-expandable-field-groups-field-group1-label1': '',
+    'form-expandable-field-groups-field-group1-label2': '',
+    'form-expandable-field-groups-field-group2-label1': '',
+    'form-expandable-field-groups-field-group2-label2': '',
+    'form-expandable-field-groups-field-group3-label1': '',
+    'form-expandable-field-groups-field-group3-label2': '',
+    'form-expandable-field-groupsform-expandable-field-groups-field-group1-label1': '',
+    'form-expandable-field-groupsform-expandable-field-groups-field-group1-label2': '',
+    'form-expandable-field-group2-label1': '',
+    'form-expandable-field-group2-label2': '',
+    'form-expandable-field-group3-label1': '',
+    'form-expandable-field-group3-label2': '',
+    'form-expandable-field-groups-field-group7-label1': '',
```

Comment:
Same idea for these... it looks like in the demo there's no "Field group 7", "Field Group 8", or "Field group 10"  so it's tricky to understand which inputs these are referring to. 

We could have something like `field-group3-nested-field-group1-label1` here - WDYT?